facial_box_dim[] bounding_box_array
